text/0037-ternary-numeric-conversion.md

```diff
@@ -0,0 +1,54 @@
++ Feature name: `ternary-numeric-conversion`
++ Start date: 2020-06-24
++ RFC PR: [iotaledger/bee-rfcs#37](https://github.com/iotaledger/bee-rfcs/pull/37)
++ Bee issue: [iotaledger/bee#93](https://github.com/iotaledger/bee/issues/93)
+
+# Summary
+
+Add a way to convert to and from numeric binary types for ternary types.
+
+# Motivation
+
+Conversion between binary and ternary is often useful and so a good solution to
+this is required.
+
+For example, this crate permits the following conversions.
```

Comment:
```suggestion
For example, this feature permits the following conversions.
```
I'd just say feature (or any other similar word) since this is what's introduced by this RFC.
That's my last nit, ready to be approved :)